### DISCUSSION TITLE
Glossary wanted

### DISCUSSION THREAD

**Discussion by Thomas-U** [vincode-io/Zavala#91] [General]
As a rookie in Outline software, it would be helpful for me to have a glossary explaining Topic and Note levels and their relationship and behaviors. And: What are Backlinks in Zavala?

Also: I have a hard time understanding what role OPML files play here.

**Comment by vincode-io** [comment]
We should probably write a help book or something to help people new to the outliner world. In the mean time wikipedia has an [article on outliners](https://en.wikipedia.org/wiki/Outliner) that can help you understand how they are structured better.

Backlinks are a newer feature to outliners. Say I link from Outline A to Outline B. Outline B will now automatically get a footnote link back to Outline A without doing anything. This is very useful if you create a network of Outlines. Backlinks are a feature you can ignore until you feel like you have a use for them.

OPML files aren't only for storing RSS Feed lists. The type of OPML files that we use for NetNewsWire are a subset of the actual [OPML spec](http://dev.opml.org/spec2.html). In fact, OPML stands for "Outline Processor Markup Language". It is the standard format that outlines are stored in.

**Reply by Thomas-U** [reply]
Thanks, Maurice, for the thorough answer!
